x-pack/plugins/observability/server/ui_settings.ts

```diff
@@ -215,7 +215,7 @@ export const uiSettings: Record<string, UiSettings> = {
     name: i18n.translate('xpack.observability.enableInfrastructureHostsView', {
       defaultMessage: 'Infrastructure Hosts view',
     }),
-    value: false,
+    value: true,
```

Comment:
So this means enabling Host view by default, right?

Reply:
yes